src/components/SingleArticle.jsx

```diff
@@ -15,9 +17,25 @@ const SingleArticle = () => {
         .then((response) => {
             const {article} = response.data
             setSingleArticle(article)
+        })
+        .catch((err) => {
+            return (
+                <p>Sorry, something went wrong, please try again</p>
+            )
         });
     },[])
+    
+    const voteUp = () => {
+        const singleArticleCopy = {...singleArticle, votes: singleArticle.votes + 1}
+        setSingleArticle(singleArticleCopy);
+        increaseVotes(article_id)
```

Comment:
At the moment we don't have anything to tell the user if their vote has been unsuccessful. (If you put your network to offline in your console, it'll still let you vote) Could you add something to give the user feedback if it's been unsuccessful please? The error handling part of these notes is really helpful for this...
https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering